internal/archive/testdata/chisel-test.asc

```diff
@@ -0,0 +1,82 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
We're 100% sure this is a one-off test key? :-)

Reply:
I was 99% sure so I've just checked and I'm now 99.9% sure it's one-off test key ;-)
<details>

```
% gpg --dearmor <./internal/archive/testdata/chisel-test.asc | gpg --list-packets 
# off=0 ctb=95 tag=5 hlen=3 plen=1368
:secret key packet:
	version 4, algo 1, created 1697568737, expires 0
	pkey[0]: [3072 bits]
	pkey[1]: [17 bits]
	skey[2]: [3065 bits]
	skey[3]: [1536 bits]
	skey[4]: [1536 bits]
	skey[5]: [1536 bits]
	checksum: db6b
	keyid: E57B791E5AEF3869
# off=1371 ctb=b4 tag=13 hlen=2 plen=37
:user ID packet: "Chisel Test <chisel-test@example.com>"
# off=1410 ctb=89 tag=2 hlen=3 plen=468
:signature packet: algo 1, keyid E57B791E5AEF3869
	version 4, created 1697568737, md5len 0, sigclass 0x13
	digest algo 8, begin of digest 8c ef
	hashed subpkt 33 len 21 (issuer fpr v4 4C951A223187332DF8E4B0FAE57B791E5AEF3869)
	hashed subpkt 2 len 4 (sig created 2023-10-17)
	hashed subpkt 27 len 1 (key flags: 03)
	hashed subpkt 9 len 4 (key expires after 2y0d0h0m)
	hashed subpkt 11 len 4 (pref-sym-algos: 9 8 7 2)
	hashed subpkt 21 len 5 (pref-hash-algos: 10 9 8 11 2)
	hashed subpkt 22 len 3 (pref-zip-algos: 2 3 1)
	hashed subpkt 30 len 1 (features: 01)
	hashed subpkt 23 len 1 (keyserver preferences: 80)
	subpkt 16 len 8 (issuer key ID E57B791E5AEF3869)
	data: [3070 bits]
# off=1881 ctb=9d tag=7 hlen=3 plen=1368
:secret sub key packet:
	version 4, algo 1, created 1697568737, expires 0
	pkey[0]: [3072 bits]
	pkey[1]: [17 bits]
	skey[2]: [3070 bits]
	skey[3]: [1536 bits]
	skey[4]: [1536 bits]
	skey[5]: [1535 bits]
	checksum: e197
	keyid: E100EB72E58B0E0C
# off=3252 ctb=89 tag=2 hlen=3 plen=444
:signature packet: algo 1, keyid E57B791E5AEF3869
	version 4, created 1697568737, md5len 0, sigclass 0x18
	digest algo 8, begin of digest 4b 01
	hashed subpkt 33 len 21 (issuer fpr v4 4C951A223187332DF8E4B0FAE57B791E5AEF3869)
	hashed subpkt 2 len 4 (sig created 2023-10-17)
	hashed subpkt 27 len 1 (key flags: 0C)
	hashed subpkt 9 len 4 (key expires after 2y0d0h0m)
	subpkt 16 len 8 (issuer key ID E57B791E5AEF3869)
	data: [3071 bits]
```

</details>

Reply:
I'm going to split commit containing this change out of this PR for CI to pass.